src/app/pages/client/inbox/Invites.tsx

```diff
@@ -102,6 +108,12 @@ function InviteCard({ room, userId, direct, compact, onNavigate }: InviteCardPro
         <Box grow="Yes">
           <Text size="T200" priority="300" truncate>
             Invited by <b>{senderName}</b>
+            {serverName && (
+              <>
+                {' on '}
+                <b>{serverName}</b>
+              </>
+            )}
```

Comment:
I think displaying only `serverName` wont be sufficient here as we only display `display_name` and the username part could be anything. maybe we should show the entire userId as "Invited by __name__ (@ name:server.name)"

Reply:
sure this depends just on how much we want to show here, so I can change it to

```suggestion
            {serverName && (
              <>
                {' ('}
                <b>{senderId}</b>
                {')'}
              </>
            )}
```

as well.

Could look a bit more cluttered now, depends on the trade of between form and function that you want to take here. Perhaps we should make this part a bit lighter.

Feel free to just apply the suggestion if you prefer this way.